test/test-framework/pkg/controller/memcachedrs/memcachedrs_controller.go

```diff
@@ -122,8 +122,11 @@ func (r *ReconcileMemcachedRS) Reconcile(request reconcile.Request) (reconcile.R
 	err = r.client.Get(context.TODO(), types.NamespacedName{Name: memcachedrs.Name, Namespace: memcachedrs.Namespace}, found)
 	if err != nil && errors.IsNotFound(err) {
 		// Define a new replicaSet
-		dep := r.replicaSetForMemcached(memcachedrs)
-		reqLogger.Info("Creating a new ReplicaSet", "ReplicaSet.Namespace", dep.Namespace, "ReplicaSet.Name", dep.Name)
+		dep, err := r.replicaSetForMemcached(memcachedrs)
+		if err != nil {
+			reqLogger.Info("Creating a new ReplicaSet", "ReplicaSet.Namespace", dep.Namespace, "ReplicaSet.Name", dep.Name)
```

Comment:
Shouldn't this be after line 129 since we only create the replicaset if there is no error here.

Reply:
actually it should be on top. good catcher, I do not know how/why it was moved.